src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -1556,4 +1561,22 @@ private function inVendors($path)
 
         return false;
     }
+
+    private function isInternalType(string $type): bool
+    {
+        $internalTypes = array(
+            'int',
+            'float',
+            'string',
+            'bool',
+            'resource',
+            'object',
+            'array',
+            'null',
+            'callable',
+            'iterable',
```

Comment:
I'd suggest to add "mixed" to the list: even if it's not an official one, it's pretty common in phpdoc so might be OK to use it here also

Reply:
After some reflections I agree with you :)